Microsoft/WebTemplateStudio
# Update submodule to allow adding casing for params

This allows to add casings for parameters, for more info on config see:
https://github.com/microsoft/CoreTemplateStudio/issues/144

edited by Tanya:
**How to test:**
1. generate a angular project with at least two pages
2. open the project in vscode and nav to "app-routing.module.ts" file
3. you should see as home url is kebab case (like the following)
![home-url](https://user-images.githubusercontent.com/12874317/59630363-f9941a00-90f9-11e9-8096-a0f68cc45390.PNG)
